icefoganalytics/elcc-data-management
# Migrate The `centre_funding_period` and `funding_period` Tables From Knex to Sequelize

Fixes https://github.com/icefoganalytics/elcc-data-management/issues/20

# Context

Migrate some more code to using Sequelize models instead of Knex queries.
Migrate the `centre_funding_period` and `funding_period` tables.

# Testing Instructions

1. Boot the app via `dev boot`.
2. Log in at http://localhost:8080/dashboard.
3. Go to http://localhost:8080/administration/funding-periods. Check that the page does crash.
> I don't have any data in my database, so not sure if it renders correctly. I fix that soon hopefully.
